templates/home_page.hbs

```diff
@@ -50,6 +64,47 @@
                 {{#is id 7156857928343}}
                 <img src="{{asset 'license_manager.svg'}}" class="custom-img-size-50" />
                 {{/is}}
+              </div>
+              <!-- End -->
+              <div class="custom-blocks-item-title-container">
+                <h2 class="blocks-item-title">{{name}}</h3>
+                  <p class="custom-blocks-item-p">{{description}}</p>
+              </div>
+              <a class="custom-blocks-item-browse-button" href='{{url}}'>{{dc 'text_browse'}}...</a>
+              </a>
+          </li>
+          {{/isnt}}
+          {{/isnt}}
+          {{/isnt}}
+          {{/isnt}}
+          {{/isnt}}
+          {{/isnt}}
+          {{/each}}
+        </ul>
+        <h1 class="geo-medium" style="font-size: 40px">Developer Products</h1>
```

Comment:
Translations for this as well.

Reply:
Placed this as {{dc 'developer_products'}} for now.